src/empty-placeholder.ts

```diff
@@ -0,0 +1,70 @@
+import { css, html, LitElement, CSSResultGroup } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import noSearchTermIcon from './assets/img/icons/no-search-term';
+import noSearchResultIcon from './assets/img/icons/no-search-result';
+
+@customElement('empty-placeholder')
+export class EmptyPlaceholder extends LitElement {
+  @property({ type: String }) placeholderType?: string;
+
+  @property({ type: Boolean }) mobileView?: false;
+
+  render() {
+    return html`
+      ${this.placeholderType === 'no-search-term'
```

Comment:
Strong suggestion: handle the `null` case here by returning `nothing`.

Remember: design component for use independent of current context.